Client/Client App/Client App/SearchForm.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Client_App.Classes;
```

Comment:
I think this file was staged by mistake since it's on the client side so please unstage it